roop/state.py

```diff
@@ -0,0 +1,64 @@
+import os
+
+from roop import utilities
+
+PROCESSED_PREFIX = '_'
```

Comment:
This should be None by default changed by the processor on the fly

Reply:
I don't get it. I introduced a constant, because it is the right and obvious decision here.